flixel/FlxCamera.hx

```diff
@@ -120,30 +120,30 @@ class FlxCamera extends FlxBasic
 	 */
 	public var scroll:FlxPoint;
 	
-	#if FLX_RENDER_BLIT
-	/**
-	 * The actual bitmap data of the camera display itself.
-	 */
-	public var buffer:BitmapData;
-	/**
-	 * Whether checkResize checks if the camera dimensions have changed to update the buffer dimensions.
-	 */
-	public var regen:Bool = false;
-	#end
+	//start renderBlit
```

Comment:
Those aren't intended to be final. Those are just placeholders to remind myself where the original sections were in case we need to add metadata or whatever to hide some of these from completion.